erpnext/modules.txt

```diff
@@ -20,4 +20,5 @@ Healthcare
 Restaurant
 Agriculture
 ERPNext Integrations
-Non Profit
\ No newline at end of file
+Non Profit
+Shopping Cart Coupon Code
```

Comment:
I think no need to add one more module. All the features can be added under already existing `Shopping Cart` module